components/scream/src/control/surface_coupling.hpp

```diff
@@ -138,10 +153,13 @@ class SurfaceCoupling {
 
   field_mgr_ptr m_field_mgr;
 
-  int           m_num_imports;
+  int           m_num_scream_imports;
+  int           m_num_cpl_imports;
   int           m_num_exports;
+  int           m_num_custom_exports;
```

Comment:
It seems that this var is never set/updated/used?

Reply:
Yep, this got added back in all the rebases.. I'll remove.